app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  require "will_paginate/array"
+  PAGE_SIZE = 4
+
+  def index
+    page = (params[:page] || '1').to_i
+
+    client = Tumblr::Client.new({
+      :consumer_key => ENV['TUMBLR_CONSUMER_KEY'],
+      :consumer_secret => ENV['TUMBLR_CONSUMER_SECRET'],
+      :oauth_token => ENV['TUMBLR_OAUTH_TOKEN'],
+      :oauth_token_secret => ENV['TUMBLR_OAUTH_TOKEN_SECRET']
+    })
+
+    result = client.posts(
+      'heatseeknyc.tumblr.com',
```

Comment:
Sure, why not. Or we could use a class constant and hoist it to the top of the file. That might be enough seeing as we don't need to change it based on the environment.